Makefile

```diff
@@ -130,7 +130,7 @@ bin/golangci-lint:
 
 GOVERTER ?= ${PROJECT_BIN}/goverter
 bin/goverter:
-	GOBIN=$(PROJECT_PATH)/bin ${GO} install github.com/jmattheis/goverter/cmd/goverter@v1.1.1
+	GOBIN=$(PROJECT_PATH)/bin ${GO} install github.com/jmattheis/goverter/cmd/goverter@v1.4.1
```

Comment:
bumps goverter 👍

Reply:
we should eventually look into avoiding specifying the version (it was required for previous work by @lampajr during some ad-interim) ... would you mind @isinyaaa logging an issue/jira to eventually change this to use a latest or some version specified in gomod if it's possible to just consolidate the version please? wdyt? (in a subsequent task of course)

Reply:
sure can look into it. I agree the Makefile is a rather "forgotten" place to specify this, as neither us nor dependabot will keep it up to date.